test/Microsoft.AspNet.Routing.Tests/Constraints/ConstraintsTestHelper.cs

```diff
@@ -28,6 +29,15 @@ public static bool TestConstraint(IRouteConstraint constraint, object value, Act
             var routeDirection = RouteDirection.IncomingRequest;
             return constraint.Match(context.Object, route, parameterName, values, routeDirection);
         }
+
+        private static IOptions<RouteOptions> GetIOptionsWithRouteOptions(bool lowerCaseUrls = true)
```

Comment:
`GetIOptionsWithRouteOptions` - We don't generally put the `I` in a name when referring to something in a variable. Just simplify it to `GetOptions`.